### PR TITLE
Add MainNav [DAH-748]

### PR DESCRIPTION
**Pull request by software-project**
Resolves https://sfgovdt.jira.com/browse/DAH-748

### CODE REVIEW

File: app/javascript/components/MainNav.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+      >
+        {t("nav.getAssistance")}
+      </a>
+      <UserNav
+        signedIn={!!profile}
+        signOut={() => {
+          setSiteAlertMessage(t(`signIn.signedOutSuccessfully`), "notice")
+          // await router.push("/sign-in")
+          signOut()
+          window.scrollTo(0, 0)
```
Comment (james-wills-sf) [inline app/javascript/components/MainNav.tsx:54]: Let's keep the redirect to the SignIn path

```suggestion
          signOut()
          // TODO: convert this to use react router when SPA routing is added
          window.location.href = getSignInPath(currentPath)
```

**Review by james-wills-sf — COMMENTED** [review]
@software-project QA Mostly looks good! We just cant check the sign in link until the next 10 stuff settles.

Have a few comments and I'd like to re-qa the redirect to the sign-in page after sign out is clicked.

**Review by akegan — APPROVED** [review]
Did QA, including checking the redirect to the sign-in page when you're logged in.

**Comment by akegan** [comment]
also feel free to ignore the lighthouse issues for now. I just disabled it but not sure how to get it to stop trying to prevent merge on this PR

**Pull request merged**